api-report/odsp-driver-definitions.api.md

```diff
@@ -176,22 +177,62 @@ export interface OdspResourceTokenFetchOptions extends TokenFetchOptions {
     siteUrl: string;
 }
 
+// @public
+export interface ShareLink {
```

Comment:
Lets call this "ShareLinkType2"/"ShareLinkMetadata"/"ShareLinkInfo" as the one before was ShareLinkTypes since this interface does not contain the actual sharelink.